tests/airgap/e2e/suite_test.go

```diff
@@ -77,6 +80,9 @@ func TestE2E(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	auditScannerVersion = os.Getenv("AUDIT_SCANNER_VERSION")
```

Comment:
In [helm chart](https://github.com/kubewarden/helm-charts/pull/697/files#diff-511d4dc00f327f8e55d0a4eee2091a17f0117253edc250c5549a242d95714521R100) workflow you use lower case to export env:

```
echo "audit_scanner_version=${AUDIT_SCANNER_VERSION}" >> ${GITHUB_OUTPUT}
```

here you use upper case `AUDIT_SCANNER_VERSION` to get it. Does this work?